src/components/FullHeader/TrialButton.js

```diff
@@ -42,7 +45,7 @@ const buttonWithMessage = ({ daysLeft, redirectURL, trialMessage }) => (
 
 const TrialButton = props => {
 	const {
-		cluster, clusterDaysLeft, daysLeft, isCluster, currentApp, trialMessage, user: { apps },
+		cluster, clusterDaysLeft, daysLeft, isCluster, currentApp, trialMessage,clusterDetails,deployment, user: { apps },
```

Comment:
Just minor thing, space is missing between `, clusterDetails, deployment`
Recommended to use `prettier` plugin with your editor.

Reply:
Sure. I have prettier already integrated in VS code, not sure how it got missed out

Reply:
I tested. Seems like prettier is showing no errors but yet does not indent properly through that line when I hit save btn

Reply:
You can use following settings fro vscode,
```
"editor.formatOnSave": true,
    "[javascript]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    },
```

or you can run `yarn format`

Reply:
Yeah actually I got the issue. It was `// prettier-ignore` which was placed here in this line @lakhansamani